pyanaconda/modules/common/__init__.py

```diff
@@ -31,3 +31,10 @@ def init():
     import locale
     from pyanaconda.core.constants import DEFAULT_LANG
     locale.setlocale(locale.LC_ALL, DEFAULT_LANG)
+
+    from pyanaconda.core.configuration.anaconda import conf
+    # The configuration is validated on the first import.
```

Comment:
Please move this comment above the line.

Reply:
Fixed.